packages/datadog-plugin-express/test/index.spec.js

```diff
@@ -1349,10 +1349,8 @@ describe('Plugin', () => {
               expect(spans[0]).to.have.property('resource', 'GET')
               expect(spans[0].meta).to.have.property('http.status_code', '404')
               expect(spans[0].meta).to.have.property('component', 'express')
-              expect(spans[0].meta).to.not.have.property('http.route')
-
-              done()
-            })
+              expect(spans[0].meta).to.have.property('http.route', '')
```

Comment:
Does it make sense to report this when empty? If there is a use case for that in the general case it's fine, but our own auto-instrumentation should not be reporting empty tags as these will show up in the UI.